### ISSUE TITLE
search field does not clear

### ISSUE THREAD

**Issue by evan4** [vueform/multiselect#99]
### Version
* Vue version: 3 

### Description
Hello, I use option :searchable="true" It is work, but when I click on item of list , selected word does not change. It is still  same letter what I wrote in search field In example the same behavior. Is it possible to replace my search word to item from list?
It is a bug in your demo also

**Comment by Libertie** [comment]
Yes, this is very important to usability. Without the selected option's label displaying, it appears as though nothing has been selected.

**Comment by Libertie** [comment]
As a work around, I am manually clearing the search field with `clearSearch()`, which causes the selected label to appear. I believe that this should be default behavior.

```
<template>
  <Multiselect
    ref="multiselect"
    @select="$refs.multiselect.clearSearch()"
    ...
  />
</template>
```

**Comment by adamberecz** [comment]
Thanks for reporting. Fixed in `2.1.1`

**Comment by tanaki** [comment]
It doesn't work for me on `2.3.3`. It looks like the fix is gone on the source file ?
